### PR TITLE
Add Litecoin

### PR DESCRIPTION
**Pull request by hellresistor**
My contribution
 - Add Litecoin
 - Update Timers for coins
 - Update Block Reward coins

### CODE REVIEW

File: js/modules/rollercoin.js [hunk 1]
```diff
@@ -42,23 +42,30 @@ export const blockReward = {
...
+        dailyReward: 0.1,
+        timePerBlock: 680,
         name: "Solana",
         ticker: "SOL",
         icon: "sol.svg"
     },
     "RST": {
         dailyReward: 150,
-        timePerBlock: 600,
+        timePerBlock: 1890,
```
Comment (Lmendev) [inline js/modules/rollercoin.js:53]: now RST timePerBlock is `902` can you update it please?

File: es/index.html [hunk 2]
```diff
@@ -84,6 +84,7 @@ <h2>Calculator</h2>
                                 <option value="RLT">RLT</option>
                                 <option value="RST">RST</option>
                                 <option value="TRX">TRX</option>
+                                <option value="LTC">LTC</option>
```
Comment (Lmendev) [inline es/index.html:87]: this line must also be included on `/index.html` for english version

File: es/pages/best-coins-to-mine.html [hunk 3]
```diff
@@ -163,6 +163,22 @@ <h2></h2>
                         </div>
                     </div>
 
+                    <div class="form-group">
```
Comment (Lmendev) [inline es/pages/best-coins-to-mine.html:166]: this too, we need to add it in `/pages/best-coins-to-mine.html` with special care of `src` path property `src="../img/ltc.svg"`

File: js/modules/rollercoin.js [hunk 4]
```diff
@@ -42,23 +42,30 @@ export const blockReward = {
...
-        timePerBlock: 600,
+        dailyReward: 60,
+        timePerBlock: 660,
         name: "Tron",
         ticker: "tron",
         icon: "trx.svg"
     },
+    "LTC": {
+        dailyReward: 0.015,
+        timePerBlock: 1848,
```
Comment (Lmendev) [inline js/modules/rollercoin.js:66]: it's `600` now :)

**Review by Lmendev — CHANGES_REQUESTED** [review]
great job @hellresistor thanks!
leave you some comments 
let me know when you can review them

I'm going to change it to draft, when it's ready we'll reopen the PR ;)

**Comment by hellresistor** [comment]
working on that :) thank you

File: es/pages/best-coins-to-mine.html (hunk 3 above)
Comment (hellresistor) [reply es/pages/best-coins-to-mine.html:166]: done

File: js/modules/rollercoin.js (hunk 4 above)
Comment (hellresistor) [reply js/modules/rollercoin.js:66]: its hard keep value updated ^^

Comment (hellresistor) [reply js/modules/rollercoin.js:66]: LTC TRX are changed time again x.x what should we put here?

Comment (hellresistor) [reply js/modules/rollercoin.js:66]: maybe possible apply an aside button to open a "config window" where is able to insert manually the timeblock minutes ? :O when empty set the "defaults" :O

**Review by hellresistor — COMMENTED** [review]
Hope all good :)

File: js/modules/rollercoin.js (hunk 4 above)
Comment (Lmendev) [reply js/modules/rollercoin.js:66]: that sounds good to me :) can be a separate pull request
this file should be for defaults

**Review by Lmendev — APPROVED** [review]
🚀

**Pull request merged**